src/Bindings/ManualBindings.cpp

```diff
@@ -3743,6 +3803,7 @@ void cManualBindings::Bind(lua_State * tolua_S)
 			tolua_constant(tolua_S, "MAX_VIEW_DISTANCE", cClientHandle::MAX_VIEW_DISTANCE);
 			tolua_constant(tolua_S, "MIN_VIEW_DISTANCE", cClientHandle::MIN_VIEW_DISTANCE);
 			tolua_function(tolua_S, "SendPluginMessage", tolua_cClientHandle_SendPluginMessage);
+			tolua_function(tolua_S, "GetForgeMods", tolua_cClientHandle_GetForgeMods);
```

Comment:
Alpha-sort the (two) functions. Perhaps add an empty line to separate the functions from the constants.